### ISSUE TITLE
Image alt tag for accessibility

### ISSUE THREAD

**Issue by diedrick** [bplmaps/argomaps#47]
In Timeline Eras:
https://argo.stage.diedrick.com/admin/timeline_eras.php/1

Can we add alternative text describing the image? I've added the field.
![image](https://user-images.githubusercontent.com/9010288/217306189-19109824-71b3-42ca-b18f-e0cb1c7d0c3c.png)

**Comment by almontg** [comment]
Yep! I also need to add alt text for all of the images in the people articles (right now some have and some don't). Will add to my list.
